src/Postal.AspNetCore/HtmlExtensions.cs

```diff
@@ -26,17 +26,15 @@ public static class HtmlExtensions
         /// <returns>An HTML &lt;img&gt; tag.</returns>
         public static IHtmlContent EmbedImage(this IHtmlHelper html, string imagePathOrUrl, string alt = "")
         {
-            if (string.IsNullOrWhiteSpace(imagePathOrUrl)) throw new ArgumentException("Path or URL required", "imagePathOrUrl");
-
+            if (string.IsNullOrWhiteSpace(imagePathOrUrl)) throw new ArgumentException("Path or URL required", "imagePathOrUrl");            
             if (IsFileName(imagePathOrUrl))
             {
-                var hosting = html.ViewContext.HttpContext.RequestServices.GetService<IHostingEnvironment>();
-                string webRootPath = hosting.WebRootPath;
-                imagePathOrUrl = System.IO.Path.Combine(webRootPath, imagePathOrUrl);
+                var hosting = html.ViewContext.HttpContext.RequestServices.GetService<IHostingEnvironment>();                
+                imagePathOrUrl = hosting.WebRootPath + "\\"+ imagePathOrUrl.Replace("/","\\");
             }
-            var imageEmbedder = (ImageEmbedder)html.ViewData[ImageEmbedder.ViewDataKey];
+            var imageEmbedder = (ImageEmbedder)html.ViewData["Postal.ImageEmbedder"];
```

Comment:
Could this use the static string instead of literal string?

Reply:
yes, I'll submit a new pull request.